documentation/src/docs/reader-group-design.md

```diff
@@ -11,83 +11,85 @@ You may obtain a copy of the License at
 
 _To be updated to reflect recent implementation details (05/07/2017)_
 
-## Motivation 
-A set of Readers can be grouped together in order that the set of Events in a Stream can be read in parallel.  The grouping of Readers is called a Readers Group.  Pravega guarantees that each Event in the Stream is read by exactly one Reader in the Readers Group. 
+## Motivation
+A set of Readers can be grouped together in order that the set of Events in a Stream can be read in parallel. This grouping of Readers is called a Reader Group. Pravega guarantees that each Event in the Stream is read by exactly one Reader in the Reader Group.
```

Comment:
My understanding of the pattern followed for documentation is as follows.
a. If we are referring to a concept then the naming convention is as `ReaderGroup` i.e. within \` \`
b. if we are talking about a stream in general term then it can start with lower case and without \` \`

Hence we could consistently name Reader Group as `ReaderGroup` , similarly for readers based on context.

Reply:
Hi @shrids ,
will go with the terminology document. In this situation we refer as concept, so no changes required!